### ISSUE TITLE
Random scrolling on Firefox 41

### ISSUE THREAD

**Issue by ofsajd** [alvarotrigo/fullPage.js#1534]
hi!

when i'm scrolling up page which uses fullpage.js script, sometimes page scrolls down, sometimes scrolls up, it's totally randomly. error also can be seen on demo page (alvarotrigo.com/fullPage/). i'm using touchpad on macbook with yosemite.

**Comment by alvarotrigo** [comment]
Are you able to reproduce it in Windows as well?

Can you be more specific about what steps we have to follow to reproduce the issue? 
I believe you don't just open the [demo page](http://alvarotrigo.com/fullPage/) and wait for it to scroll randomly. Right?

**Comment by ofsajd** [comment]
Here is the video which shows how it looks like: http://1drv.ms/1QzYhtN
I'm just scrolling all the way down by macbook touchpad

I'll test it in windows tomorrow

**Comment by alvarotrigo** [comment]
The video doesn't show if you are scrolling or not. Were you? And if so, at which moment?

**Comment by ofsajd** [comment]
Yes, on this video i'm only scrolling in one direction (down), but as you can see sections moves randomly

**Comment by alvarotrigo** [comment]
I can not reproduce the problem in Browserstack using Yosemite and Firefox 41. 
I'll try it from home later, but it sounds like the problem might be in your side. Maybe your trackpad? 

Can you enter [here}(http://jsfiddle.net/bpwydfne/1/) and let me know:

- Do you see any messages on the bottom right frame without using your trackpad at all? 
- Now, click on the right side frame to focus it and **scroll down only**. Do you see any green mesages saying "You scrolled up"?
- Now do the opposite, **scroll up only**. Do you see any red messages saying "You scrolled down"?

**Comment by ofsajd** [comment]
Firefox: 
1 / no
2 / yes
3 / yes

but in Chrome everything is okay:
1 / no
2 / no 
3 / no

I tested it on two macbooks

**Comment by alvarotrigo** [comment]
> Firefox: 
> 1 / no
> 2 / yes
> 3 / yes

Right, let see...
Can you answer the same questions for [this other link](http://jsfiddle.net/bpwydfne/2/)? I got rid of now of old Firefox mouse wheel detection.

**Comment by ofsajd** [comment]
in second version: 
Firefox & Chrome:
1 / no
2 / no
3 / no

:)

**Comment by alvarotrigo** [comment]
Good. Problem detected :)

Your issue is the same causing [this other problem](https://github.com/alvarotrigo/fullPage.js/issues/1510) and was introduced time ago by [an external pull request](https://github.com/alvarotrigo/fullPage.js/issues/1244).

I will solve the issue for the next release, but if you want to get rid of the problem meanwhile, just remove the [line 2255](https://github.com/alvarotrigo/fullPage.js/blob/master/jquery.fullPage.js#L2255) of the latest fullpage.js version:

```javascript
document.removeEventListener('DOMMouseScroll', MouseWheelHandler, false); //old Firefox
```

**Comment by ofsajd** [comment]
thank a lot! :)

**Comment by alvarotrigo** [comment]
Can you do me a favour and let me know if [this example](http://jsfiddle.net/bpwydfne/6/) works ok for you as well in the 3 cases?

If that works, that's probably the solution I'll choose for the next release.

**Comment by ofsajd** [comment]
yes, it works :)

**Comment by alvarotrigo** [comment]
Thanks for it!

**Comment by alvarotrigo** [comment]
Solved in 2.7.2
